packages/cadl-typescript/CONTRIBUTING.md

```diff
@@ -0,0 +1,127 @@
+# Introduction
+
+This library is the cadl typescript emitter for Rest Level Client. It take cadl as input, transform it into RLCModel, then call rlc-common library to generate the RLC code.
+
+On a high level, the entire Rest Level Client generation process would be:
```

Comment:
Maybe this introduction could be put into readme.md and let's focus on how to contribute in this spec.

Reply:
updated.